### PR TITLE
Windows Authentication Parser Update (VIM and IM)

### PR DESCRIPTION
**Pull request by sparkmark**
Required items, please complete
   
   Change(s):
   - Update VIM Parser to include new event ID's

   Reason for Change(s):
   - Update parser for better coverage

   Version Updated:
   - N/A

   Testing Completed:
   - Testing completed in my lab

   Checked that the validations are passing and have addressed any issues that are present:

**IM File Schema Validation**

(0) Error: type mismatch for column [EventOriginalType]. It is currently int and should be string --> This is an error of the original parser. 

**IM File Data Validation**
(0) Error: 1 invalid value(s) (up to 10 listed) in 66 records (66.0%) for field [EventSubType] of type [Enumerated]: ["Network"] (Schema:Authentication) --> This is an error of the original parser. 

(0) Error: type mismatch for column [EventOriginalType]. It is currently [int] and should be [string] (Schema:Authentication) --> This is an error of the original parser. 

**VIM File Schema Validation**

(0) Error: type mismatch for column [EventOriginalType]. It is currently int and should be string --> This is an error of the original parser. 

**VIM File Data Validation**

(0) Error: 1 invalid value(s) (up to 10 listed) in 100 records (100.0%) for field [EventSubType] of type [Enumerated]: ["Network"] (Schema:Authentication) --> This is an error of the original parser.

(0) Error: type mismatch for column [EventOriginalType]. It is currently [int] and should be [string] (Schema:Authentication) --> This is an error of the original parser. 

(1) Warning: Empty value in 100 records (100.0%) in mandatory field [EventSchema] (Schema:Authentication) --> This is an error of the original parser.

(2) Info: Empty value in 100 records (100.0%) in optional field [TargetSessionId] (Schema:Authentication) --> This is an error of the original parser.

### CODE REVIEW

**Comment by sparkmark** [comment]
@vakohl ... just in case this is new PR that includes both files. I have also added the results of the tester. All the errors we get are from the config of the original parser. Let me know if you want to update the original parser to fix those issues as part of this update ?

**Comment by v-atulyadav** [comment]
Hi @sparkmark,
Thank you for these commits. We will provide an update as soon as possible.

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftWindowsEvent.yaml [hunk 1]
```diff
@@ -22,6 +22,57 @@ ParserParams:
 ParserQuery: |
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftWindowsEvent.yaml:22]: can you increase the parser version to 0.2.2

Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftWindowsEvent.yaml:22]: can you please add following property under ParserName?
EquivalentBuiltInParser: _ASim_Authentication_MicrosoftWindowsEvent

For reference: https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoASA.yaml

File: Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftWindowsEvent.yaml [hunk 2]
```diff
@@ -176,5 +230,79 @@ ParserQuery: |
          User=TargetUsername
         , LogonTarget=TargetDvcHostname
         , Dvc=SrcDvcHostname
+  };
+  let WinLogin4768 = (disabled:bool=false){
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftWindowsEvent.yaml:234]: do we need this block if we add 4768 in LogonEvents variable declared above?

File: Parsers/ASimAuthentication/Parsers/vimAuthenticationMicrosoftWindowsEvent.yaml [hunk 3]
```diff
@@ -31,6 +31,57 @@ ParserParams:
 ParserQuery: |  
   let LogonEvents=dynamic([4624,4625]);
   let LogoffEvents=dynamic([4634,4647]);
+  let LogonStatus4768=datatable
```
Comment (vakohl) [inline Parsers/ASimAuthentication/Parsers/vimAuthenticationMicrosoftWindowsEvent.yaml:34]: same comments as ASIM parser applicable here.

**Comment by vakohl** [comment]
@sparkmark Thank you for your contribution. Can you please attach parser test results as file? Please follow guidelines: https://learn.microsoft.com/en-us/azure/sentinel/normalization-develop-parsers#test-results-submission-guidelines

Also, if you can attach the sample data too https://learn.microsoft.com/en-us/azure/sentinel/normalization-develop-parsers#samples-submission-guidelines

**Review by vakohl — CHANGES_REQUESTED** [review]

**Comment by v-atulyadav** [comment]
Hi @sparkmark, hope you are doing well. Just wanted to check if you got a chance to look at the suggestions shared. Please feel free to reach out to us for any queries and/or support.

**Comment by v-atulyadav** [comment]
Hi @sparkmark,
We'd appreciate it if you could let us know when your work is finished. Feel free to contact us with any queries or support.
Thanks

**Comment by vakohl** [comment]
@v-atulyadav I'm looking into past discussions internally regarding this PR. Pls wait for my update

**Comment by vakohl** [comment]
@sparkmark Wanted to check regarding this PR. Would you like us to review the changes in this PR?

**Comment by v-atulyadav** [comment]
Hi @sparkmark,
Please confirm if you would like us to review the changes in this PR as per vakohl's comment? Thanks

**Comment by v-atulyadav** [comment]
Hi @sparkmark,
We're waiting for an update on this. Thanks

**Comment by sparkmark** [comment]
> Hi @sparkmark,
> We're waiting for an update on this. Thanks

Hi, at this point in time we are not progressing with ASIM.

**Comment by v-atulyadav** [comment]
Hi @sparkmark,
We are closing this PR now if you still need support for this issue, you can re-open the PR at any time. 
If you do re-open, we simply request that you ensure the PR has response to the last request. Thank you for your cooperation. Thank you.

**Pull request closed without merging**